builtin/rebase.c

```diff
@@ -700,6 +704,8 @@ static int run_specific_rebase(struct rebase_options *opts)
 		argv_array_pushf(&child.args, "--head-name=%s",
 				 opts->head_name ?
 				 opts->head_name : "detached HEAD");
+		if (opts->ignore_whitespace)
+			argv_array_push(&child.args, "--ignore-whitespace");
```

Comment:
Why are we passing --ignore-whitespace to the builtin/rebase--interactive.c program instead of translating here to -Xignore-space-change?

However, the question is kind of moot, since this and the surrounding code no longer exist in modern versions of git.

Reply:
We are passing it as `--ignore-whitespace` since I've introduced this flag to `builtin/rebase--interactive.c` and all the parsing has been done in this file only.

Reply:
But yeah, I'll rebase and see what needs to be done.